[2025-01-27T15:39:28.891+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-27T15:39:28.907+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: executing_multiple_tasks.taskD manual__2025-01-27T15:39:26.616840+00:00 [queued]>
[2025-01-27T15:39:28.914+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: executing_multiple_tasks.taskD manual__2025-01-27T15:39:26.616840+00:00 [queued]>
[2025-01-27T15:39:28.916+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-27T15:39:28.936+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): taskD> on 2025-01-27 15:39:26.616840+00:00
[2025-01-27T15:39:28.942+0000] {standard_task_runner.py:72} INFO - Started process 2520 to run task
[2025-01-27T15:39:28.945+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'executing_multiple_tasks', 'taskD', 'manual__2025-01-27T15:39:26.616840+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/execute_multiple_tasks.py', '--cfg-path', '/tmp/tmppxuxq8nf']
[2025-01-27T15:39:28.947+0000] {standard_task_runner.py:105} INFO - Job 30: Subtask taskD
[2025-01-27T15:39:29.004+0000] {task_command.py:467} INFO - Running <TaskInstance: executing_multiple_tasks.taskD manual__2025-01-27T15:39:26.616840+00:00 [running]> on host 6eaa7309f7d4
[2025-01-27T15:39:29.082+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='executing_multiple_tasks' AIRFLOW_CTX_TASK_ID='taskD' AIRFLOW_CTX_EXECUTION_DATE='2025-01-27T15:39:26.616840+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-27T15:39:26.616840+00:00'
[2025-01-27T15:39:29.084+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-27T15:39:29.101+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-27T15:39:29.103+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', 'tmp6iil93c6.sh']
[2025-01-27T15:39:29.114+0000] {subprocess.py:99} INFO - Output:
[2025-01-27T15:39:29.116+0000] {subprocess.py:106} INFO - TASK D completed!
[2025-01-27T15:39:29.117+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-27T15:39:29.143+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-27T15:39:29.145+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=executing_multiple_tasks, task_id=taskD, run_id=manual__2025-01-27T15:39:26.616840+00:00, execution_date=20250127T153926, start_date=20250127T153928, end_date=20250127T153929
[2025-01-27T15:39:29.201+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-27T15:39:29.233+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-27T15:39:29.241+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
